tests/unit_tests/test_wave_07 - BONUS ROUND.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+
+def test_newest():
```

Comment:
Solid happy-path tests! What other tests could we add to cover edge cases around the age attribute and age operations?